amoro-core/pom.xml

```diff
@@ -140,6 +140,11 @@
             <artifactId>paimon-bundle</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.paimon</groupId>
+            <artifactId>paimon-s3</artifactId>
+        </dependency>
+
```

Comment:
I think this dependency should belond to `amoro-ams-server` module .

Reply:
ok